tests/cockpit-tasks-centosci.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: v1
+kind: List
+items:
+- kind: ReplicationController
+  metadata:
+    name: cockpit-tasks
+  spec:
+    replicas: 1
+    selector:
+      infra: cockpit-tasks
+    template:
+      metadata:
+        labels:
+          infra: cockpit-tasks
+      spec:
+        containers:
+        - name: centosci-tasks
+          image: docker.io/cockpit/tests
+          securityContext:
+            hostNetwork: false
```

Comment:
I don't think we need this on CentOS CI either - our ad-hoc pods seemed to work well without it, so I'm in favor of completely dropping the `securityContext` section.